xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/commentfield.vm

```diff
@@ -56,10 +56,6 @@
 
 #set ($fieldToDisplay = $commentObjClass.get($name))
 
-## We need to know the type of editor used for comments in order to hide/display the preview button.
-#set ($defaultEditorId = $services.edit.syntaxContent.defaultEditorId)
-<input type='hidden' name='defaultEditorId' value="$escapetool.xml($defaultEditorId)" />
```

Comment:
Not sure why you removed that one: maybe I missed it but didn't see it supported in your other changes.